substrate/bin/node/runtime/src/migrations.rs

```diff
@@ -37,71 +70,3 @@ pub mod add_contracts_registry_pallet {
 		}
 	}
 }
-
-pub mod add_sora_bridge {
```

Comment:
why delete this ?

Reply:
It's a migration, so it was needed only once for single runtime upgrade (v18 -> v19). It's no longer used anywhere, no reason to keep it.